### PR TITLE
Added text utility, navigation and list components.

### PR DESCRIPTION
**Pull request by j24m**
Added 3 new component namely text-utility, list and navigation.

### CODE REVIEW

File: Components/list.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.unordered-ordered-lists{
+    color: var(--light-color);
+    font-size: 20px;
+    margin:50px;
```
Comment (zoef-yt) [inline Components/list.css:4]: Using rem would be better.

File: main.css [hunk 2]
```diff
@@ -50,11 +50,12 @@ a:hover{
     justify-content: space-between;
 }
 .side-bar{
+    min-width:200px;
     padding:0 30px ;
-    flex: 1;
+    /* flex: 1 */
```
Comment (zoef-yt) [inline main.css:56]: Remove commented code.

File: Components/text-utility.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    line-height:50px;
+    padding:10px;
+    text-shadow: 2px 2px 5px var(--light-color);
+    /* font-family: "Dancing Script"; */
+}
+.grey-text{
+    text-shadow: none;
+    font-size:40px;
+    background-color: var(--secondary-color);
+    color:gray;
```
Comment (zoef-yt) [inline Components/text-utility.css]: Use variable for colours

File: Components/text-utility.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
+.heading-utility,.small-textutility{
+    line-height:50px;
+    padding:10px;
+    text-shadow: 2px 2px 5px var(--light-color);
+    /* font-family: "Dancing Script"; */
```
Comment (zoef-yt) [inline Components/text-utility.css]: Remove commented code.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: Components/navigation.css [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+}
+.nav-searchbar::placeholder{
+    font-size: 15px;
+    color:var(--blue-shade-color1);
+}
+.nav-home-shopnow-icons:hover{
+    color:var(--primary-color);
+    transform : scale(1.1);
+    text-shadow: 2px 2px 10px var(--light-color);
+}
```
Comment (rahulk31) [inline Components/navigation.css:53]: Suggestion:  Text Shadow can be removed to make it more generic. If required somewhere can be added then explicitly.

File: Components/navigation.css [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+.right-of-nav{
+    display: flex;
+    width:40%;
+    justify-content: space-evenly;
+    align-items: center;
+}
+.nav-logo{
+    font-family: 'Dancing Script';
+    font-size:30px;
+    text-shadow:2px 2px 5px var(--primary-color);
```
Comment (rahulk31) [inline Components/navigation.css:30]: Suggestion: Text Shadow can be removed to make it more generic. If required somewhere can be added then explicitly.

File: Components/navigation.css [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+.nav-searchbar:focus{
+    background-color: var(--cream-shade-color);
+}
+.nav-searchbar::placeholder{
+    font-size: 15px;
+    color:var(--blue-shade-color1);
+}
+.nav-home-shopnow-icons:hover{
+    color:var(--primary-color);
+    transform : scale(1.1);
```
Comment (rahulk31) [inline Components/navigation.css:51]: For hover, scale can be reduced a bit so that it looks subtle and has a provides a better UX.

**Review by rahulk31 — COMMENTED** [review]
Overall looks good to me, can be merged.

**Review by samyakshah3008 — APPROVED** [review]
Loved the work, especially Nav bar 
Looks good to merge
One suggestion: Please put the live hosted link for better review

**Pull request merged**